sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_client_manager.py

```diff
@@ -293,6 +298,7 @@ def __init__(
         replica_discovery_enabled,
         min_backoff_sec,
         max_backoff_sec,
+        load_balance: bool,
```

Comment:
```suggestion
        load_balancing_enabled: bool,
```

Reply:
Also, why is this explicitly bool here while other arguments dont have a type?

Reply:
I don't think we have it setup where private methods need to have type hints. I can remove it if you want.

Reply:
I think we should remove it - or add it for other argument. I'm fine with being consistent either way.